src/frontend/screens/Settings/components/WineVersionSelector.tsx

```diff
@@ -102,15 +128,13 @@ export default function WineVersionSelector() {
             <InfoBox text="infobox.help">
               <span>{t('help.wine.part1')}</span>
               <ul>
-                <i>
```

Comment:
this `<i>` tag here was invalid HTML

Reply:
the problem is not the tag itself, it's that `<i>` can't be a direct children of `<ul>` (and the browser is fixing that for us), I'll update this to put the `<i>` inside each `<li>`

actually, I'll check if I can do this with CSS, I don't like tags for style purposes